### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by sheikhre** [sheikhre/MI-349-FS17-741-html-final-website-bmMc9J#1]
## Final Website

@jsiarto Can you take a look at this? It's [hosted here](https://sheikhre-html-final-website.surge.sh/) and meets the following criteria:

- [x] 4 valid HTML pages (1 homepage and 3 internal pages)
- [x] Site must be styled using CSS (typography and layout)
- [x] Bootstrap can be used, but you must also include at least 10 custom CSS rules
- [x] All images and copy must be complete (no placeholder images or "lorem ipsum")
- [x] Site must be based off of your pitchboard and personas from the previous modules
- [x] Site must include 1 @media query (mobile/responsive rule)
- [x] No paths should display "index.html"--use the same method for internal pages as you did in the multiple pages module
- [x] Site is hosted on Surge

**Comment by jsiarto** [comment]
- Site just lists personas, it should be about the company not the target audience.

**Comment by sheikhre** [comment]
I updated the site to not show personas

**Comment by jsiarto** [comment]
No, the site needs to be based on the personas--it needs to be a complete site for the company in which your pitchboard is based (removing the personas is not enough). The semester work deadline is passed and we can't take any additional work.

**Comment by sheikhre** [comment]
I updated the site, which is based off the persona's instead of the actual persona page. Each persona was changed to a page which gives more information of the company. I have also included projects which the company is focusing on. I was hoping to be able to revise the work like past modules in order to pass. I'm sorry for any problems this caused, and appreciate your time. I was really hoping to receive credit for all 16 modules, which would help me out immensely.

**Comment by jsiarto** [comment]
:shipit:
